### PR TITLE
Networking Beta

### PR DESCRIPTION
**Pull request by designatednerd**
Future changes to the networking stack will get merged into this until it comes out of beta (hopefully soon). Opening a PR to get Netlify to auto-generate updated docs that can be referenced here. 

Please note: 

- If you're opening a bug about the networking beta, please prefix the bug title with `Networking Beta:` - this will help me narrow down where things are going wrong more quickly.  
- If you're using the beta, please check the `CHANGELOG` from this branch for updates.
- Updated documentation and code are available here: https://deploy-preview-1386--apollo-ios-docs.netlify.app/docs/ios/
- The code changes to the tutorial are available in the [Tutorial Repo](https://github.com/apollographql/iOSTutorial) on the `new-network-beta` branch.

### CODE REVIEW

File: Sources/Apollo/InterceptorProvider.swift [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        AutomaticPersistedQueryInterceptor(),
+        LegacyCacheWriteInterceptor(store: self.store),
+    ]
+  }
+}
+
+// MARK: - Default implementation for swift codegen
+
+
+/// The default interceptor proider for code generated with Swift Codegen™
```
Comment (cswelin) [inline Sources/Apollo/InterceptorProvider.swift:59]:
```suggestion
/// The default interceptor provider for code generated with Swift Codegen™
```

Comment (designatednerd) [reply Sources/Apollo/InterceptorProvider.swift:59]: 🤦‍♀️ Good catch

**Comment by heyzooi** [comment]
hi @designatednerd, I'm wondering if you have an estimate release date for a new version including this PR?

**Comment by designatednerd** [comment]
@heyzool - Hoping for monday evening if nobody finds any showstoppers

**Pull request merged**